superset-frontend/plugins/plugin-chart-ol/src/util/geometryUtil.ts

```diff
@@ -55,8 +34,9 @@ export const convertLocationsToFeatures = (locations: Array<String>) => {
  *
  * @returns The coordinate
  */
-export const getCoordinateFromLocation = (location: string) => {
-  const geom: Point = new GeoJSON().readGeometry(location, {
+export const getCoordinateFromGeometry = (geometry: Geometry) => {
+  // TODO change signature so the we already get a coordinate array
```

Comment:
not clear to me what is meant here

Reply:
Removed comment which was just a leftover